tests/accelerators/test_accelerator_connector.py

```diff
@@ -506,6 +506,26 @@ def test_accelerator_cpu(_):
         trainer = Trainer(accelerator="cpu", gpus=1)
 
 
+@mock.patch("torch.cuda.is_available", return_value=True)
+@mock.patch("torch.cuda.device_count", return_value=2)
+@pytest.mark.parametrize("gpu_count", (["0"], [0, "1"], ["GPU"], [["0", "1"], [0, 1]]))
+def test_accelererator_invalid_type_gpus(mock_is_available, mock_device_count, gpu_count):
+    with pytest.raises(
+        MisconfigurationException, match=r".* must be an int, a string, a sequence of ints or None, but you .*"
```

Comment:
This is done automatically

```suggestion
        MisconfigurationException, match=r"must be an int, a string, a sequence of ints or None, but you"
```